skills/nasa/index.js

```diff
@@ -0,0 +1,30 @@
+const request = require('../../api/request');
+
+//put your nasa api key here:
+const key = '';
```

Comment:
Would you mind fetching the API key from the config DB please? https://github.com/patrickjquinn/P-Brain.ai/blob/master/skills/weather/index.js#L9

Reply:
I looked over the weather skill, and I think I got it working. Let me know if this is the correct way of doing it.

Reply:
Yep, looks good